client/src/components/relatedItems/Cards.jsx

```diff
@@ -59,32 +58,33 @@ class Cards extends Component {
 
   }
 
+  setNewItem () {
+    this.props.handleOverviewIdChange(this.props.id);
+  }
+
   render() {
 
     return (
       <>
       {this.state.image !== null &&
-        <div className="card" key={this.props.id} data-testid='test-id'>
-          <div className="card-image">
+        <div className="card" data-testid='test-id'>
 
-          <div className="actions">
-              <button type="button" className="card-button">
-              </button>
+          <div className="card-image">
+            <img src={this.state.image} alt='This is an image of the product as described below.' onMouseDown={this.setNewItem}/>
```

Comment:
I love the detailed description in the alt. I forgot this is used for accessibility. 😄